api/types/container/config.go

```diff
@@ -63,6 +63,7 @@ type Config struct {
 	MacAddress      string              `json:",omitempty"` // Mac Address of the container
 	OnBuild         []string            // ONBUILD metadata that were defined on the image Dockerfile
 	Labels          map[string]string   // List of labels set to this container
+	Annotations     map[string]string   // List of annotations set to this container
```

Comment:
I feel like the naming around this is too generic. If we want to provide passthroughs to the OCI runtime then naming/structure should reflect this.

Reply:
Do you have something in mind that would be a less generic naming ?

Reply:
@cpuguy83 you mean like `OciAnnotations` or  `RuntimeAnnotations` or other things like these to indicate that this belongs to `OCI spec`

Reply:
Maybe something like `RuntimeAnnotations`... but I'm curious why these configurations aren't passed down as runtime args?

Reply:
Well if we use args, this means everytime a new thing comes up, we'll need to get those changes accepted. Annotations is the most generic way to pass arguments that might not be defined yet.

Reply:
Are these annotations unique for each individual container, or for the runtime that's used? (i.e., must each container have different annotations?)

Reply:
It's per container since they will be used to differentiate containers from pods.